Documentation/api-reference.md

```diff
@@ -149,11 +154,11 @@ var mediaInfo = Media.createMediaObject("video-name",
 
 Creates an instance of the AdBreak object.
 
-| Variable Name | Description | Required |
+| Parameter | Description | Required |
 | :--- | :--- | :---: |
-| `name` | Ad break name such as pre-roll, mid-roll, and post-roll. | Yes |
-| `position` | The number position of the ad break within the content, starting with 1. | Yes |
-| `startTime` | Playhead value at the start of the ad break. | Yes |
+| `name` | The friendly name of ad break such as pre-roll, mid-roll, and post-roll | Yes |
```

Comment:
"The friendly name of **the** ad break such as pre-roll, mid-roll, **or** post-roll"